### ISSUE TITLE
Double yml syntax shouldn't be allowed

### ISSUE THREAD

**Issue by tristanbes** [ansible/ansible-lint#214]
# Issue Type
- Feature request
# Ansible and Ansible Lint details

```
ansible 2.1.2.0
ansible-lint 3.3.3
```
# Desired Behaviour

When using this (which is wrong, but it happened to me) 

```
- name: Update git config
  template: >
    src: install_from_git/gitconfig
    dest: "{{ yprox_version_dir }}/.git/config"
```

**instead of**

```
- name: Update git config
  template: 
    src: install_from_git/gitconfig
    dest: "{{ yprox_version_dir }}/.git/config"
```

It should trigger an error/warning about how the 2 argument `dest` will be ignored so the module `template` will throw an error.
# Actual Behaviour (Bug report only)

Ansible lint doesn't throw an error

**Comment by willthames** [comment]
It's not immediately obvious if there are good ways to check this.

Once the YAML is loaded it looks like:

```
[{'hosts': 'localhost',
  'tasks': [{'name': 'Update git config',
             'template': 'src: install_from_git/gitconfig dest: "{{ yprox_version_dir }}/.git/config"\n'}]}]
```

but without a deep knowledge of the template module, which ansible-lint is not set up to acquire, it's difficult to detect this bug.

**Comment by tristanbes** [comment]
It not about the specific "template" module. The syntax problem can happen in any module

Before we only wrote ansible with
```
module: >
  arg1=value1
  arg2=value2
```

and now
```
module: 
  arg1: value1
  arg2: value2
```

but if it's difficult to detect this you can close this issue :(

**Comment by willthames** [comment]
Actually this might be detectable - we expect the value of the module key (in this case `template`) to be a `dict`, not a string. There are a few exceptions to this (`command`, `shell`, `raw`) but that's not insurmountable.

**Comment by ssbarnea** [comment]
Run yamllint before ansible-lint.
